go.mod

```diff
@@ -1,15 +1,9 @@
 module github.com/mattermost/mattermost-giphy-plugin
 
-go 1.12
+go 1.13
 
 require (
-	github.com/gorilla/mux v1.7.0
-	github.com/mattermost/go-i18n v1.11.0 // indirect
-	github.com/mattermost/mattermost-server v5.12.0+incompatible
+	github.com/gorilla/mux v1.7.3
+	github.com/mattermost/mattermost-server v0.0.0-20190927121038-340287890a78
```

Comment:
What release is this?

Reply:
Using 5.16.x appears to have other problems, replaces etc.

Reply:
Not blocking for this PR, but can you please elaborate? If there are issue blocking plugin devs from using a specific version, we should investigate.